script.100fm/resources/settings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category>
+        <setting label="10000" type="number" id="channel" default="0" visible="false"/>
```

Comment:
Hi @phil65 , I do need it since it also sets the default values for these variables and without the settings file the add-on crashes.